controls/V-71989.rb

```diff
@@ -64,9 +43,10 @@
 SELINUX=enforcing
 
 A reboot is required for the changes to take effect."
+  tag "fix_id": "F-78341r2_fix"
 
-  #@todo - SELinux resource?? (https://github.com/chef/inspec/issues/534)
+  # TODO SELinux resource?? (https://github.com/chef/inspec/issues/534)
   describe command('getenforce') do
-    its('stdout'.strip) { should match(%r{^Enforcing$}) }
+    its('stdout.strip') { should match %r{^Enforcing} }
```

Comment:
This should just be a string match.

Something like `its ('stdout.strip') { should eq 'Enforcing' )`